packages/volto/src/components/manage/Blocks/Teaser/Data.jsx

```diff
@@ -70,14 +70,22 @@ const TeaserData = (props) => {
   };
 
   const refresh = () => {
-    dispatch(
-      getContent(flattenToAppURL(data.href[0]['@id']), null, `${block}-teaser`),
-    ).then((resp) => {
-      if (resp) {
-        let blockData = dataTransformer(resp, data);
-        onChangeBlock(block, blockData);
-      }
-    });
+    if (data.href?.[0]?.['@id']) {
+      dispatch(
+        getContent(
+          flattenToAppURL(data.href[0]['@id']),
+          null,
+          `${block}-teaser`,
+        ),
+      )
+        .then((resp) => {
+          if (resp) {
+            let blockData = dataTransformer(resp, data);
+            onChangeBlock(block, blockData);
+          }
+        })
+        .catch((e) => {});
```

Comment:
I would recommend not ignoring exceptions here. If there's a real problem processing the request on the server side, we don't want to hide it.

Reply:
@nileshgulia1 I agree with @davisagli  When i've seen the commit I was thinking that I would still like to have some sort of notification regarding a potential issue.
It would depend though on what type of error it get's and how should we display it.

@davisagli what do you propose? I'm thinking maybe a Toast message with an error has occurred see the console for details.
This way the editor knows that it's an issue and if needed the developer can see the error from the browser console.

Reply:
The toast first came into my mind since it's not only the external Url case but also how we are currently doing in the blocks.

Reply:
@davisagli Right now in this PR, if the target item is deleted we disable the "refresh button". The only case is if the target URL is invalid we have to handle that using a toast.

Reply:
@nileshgulia1 Generally yes. But I think there is a race condition if the editor loads the page, then the target is deleted, and then the editor tries to use the refresh button.